src/PostCampaign.php

```diff
@@ -378,13 +416,34 @@ private static function get_theme_defaults( $theme_json ) {
 		return $defaults;
 	}
 
+	public static function get_legacy_themes(): array {
+		return [
+			'default',
+			'antarctic',
+			'arctic',
+			'climate',
+			'oceans',
+			'oil',
+			'plastics',
+			'forest',
+		];
+	}
+
 	/**
 	 * Determine the css variables for a certain post.
 	 *
 	 * @param array $meta The meta containing the variable values.
+	 *
 	 * @return array The values that will be used for the css variables.
 	 */
 	public static function css_vars( array $meta ): array {
+		$is_new_theme = $meta['theme'] && ! in_array( $meta['theme'], self::get_legacy_themes(), true );
+		if ( $is_new_theme ) {
+			$themes = json_decode( get_option( 'planet4_themes', '[]' ), true );
```

Comment:
This is probably temporary to allow us to make changes to the theme on the instance. After we confirmed the new version works, we can for now include it in our code base, so that it effectively works the same as the legacy themes.